### DISCUSSION TITLE
Welcome to JParty Discussions!

### DISCUSSION THREAD

**Discussion by stuartthomas25** [stuartthomas25/JParty#2] [Announcements]
## 👋 Welcome to JParty Discussions!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by btr70** [comment]
Hi, I'm a user (not a developer) with a troubleshooting question. Can I ask that here? I hesitate to open an issue if it's just a configuration problem.

**Reply by stuartthomas25** [reply]
Hi @btr70 ! 

Of course! What is your issue?

**Reply by btr70** [reply]
Thanks. I got tripped up on the instructions in the README since `cli.py` and `JParty.spec` don't seem to exist. I've got it running now, though. Nice work, it's really slick! 😀

Is there a way to move on to the next board without finishing all the questions, or abort the current game and get back to the welcome screen? Not a big deal, just curious.

**Reply by stuartthomas25** [reply]
Ah yes, that was an issue in the README which has been fixed. Sorry about that!

Also no, there isn't a feature to skip or exit rounds currently but I am open to adding it! Feel free to file a Issue requesting this feature! 😃
